subgraphs/compound-forks/src/versions.ts

```diff
@@ -0,0 +1,17 @@
+import { Versions as VersionsInterface } from "../../../deployment/context/interface";
+
+export class VersionsClass implements VersionsInterface {
+  getSchemaVersion(): string {
+    return "2.0.1";
+  }
+
+  getSubgraphVersion(): string {
+    return "1.1.9";
+  }
+
+  getMethodologyVersion(): string {
+    return "1.0.0";
+  }
+}
+
+export const Versions = new VersionsClass();
```

Comment:
Does this change mean that all compound-forks subgraphs will have the exact same versions? Because versions.ts is in compound-forks/src, and there is no protocol-specific versions.ts file.

Reply:
@tnkrxyz I just made a PR to remove this #1361

This is an auto-generated file that helps update versions automatically based on the deployment.json version. It shoudln't be in master, but you will notice it generates each time you use messari b -d